index.js

```diff
@@ -74,4 +72,4 @@ for( var l in levels ) { ( function( prio, name ) {
 		log( prio, message, fields );
 	};
 
-} )( l, levels[ l ] ); }
+} )( l * 10, levels[ l ] ); }
```

Comment:
To make it clear that this is an internal implementation, basically.  The number is our own custom one.  I guess you could leave the number alone...